docs/topics/spiders.rst

```diff
@@ -703,6 +703,16 @@ SitemapSpider
         This would retrieve only ``entries`` modified on 2005 and the following
         years.
 
+        Entries are dict objects extracted from the sitemap document.
+        Usually, the key is the tag name and the value is the text inside it.
+
+        It's important to notice that:
+
+        - as the loc attribute is required, entries without this tag are discarded
+        - alternate links are stored in a list with the key ``alternate``
+          (see ``sitemap_alternate_links``)
+        - namespaces are removed, so lxml tags named as ``{foo}bar`` become only ``bar``
```

Comment:
Is the syntax for namespaces correct? the docs you propose used curly braces syntax `{foo}bar` while the namespaces as being tested in `test_spider.py` uses `<namespace:tagname>` syntax.

Reply:
btw, I think we lack a testcase for namespaces in test for sitemap filter.

Reply:
@dangra, when we're parsing the tags with lxml it puts the tag's namespace inside curly braces. 

`<namespace:tagname>` would be named as `{namespace}tagname`.

That's why we're doing this: https://github.com/scrapy/scrapy/blob/4306886ac6030c9312547bdbcff3d88eb89c11d0/scrapy/utils/sitemap.py#L27

Reply:
I have just added a test case to cover sitemap filter usage with alternate links. If you're satisfied with the test case, could you resolve this conversation? Otherwise, I'm open to additional comments and directions.

Reply:
I have also slightly changed docs: `{foo}bar` is now `{namespace}tagname` as you have just used. Think it will make things clearer.